src/pages/gen1/[platform]/prev/build-a-backend/storage/copy/index.mdx

```diff
@@ -9,7 +9,21 @@ export const meta = {
     'angular',
     'nextjs',
     'react',
-    'vue'
+    'vue',
+    'flutter'
+  ],
+  canonicalObjects: [
+    {
+      platforms: [
```

Comment:
Q: Is it intentional that not all JS platforms were added here? I don't fully understand the impact of this change.

Reply:
Same comment applies to files below.